docs/_static/css/custom.css

```diff
@@ -328,5 +554,27 @@ a:hover, .wy-breadcrumbs-aside a {
 .wy-nav-top > i {
   font-size: 24px;
   padding: 8px 0 0 2px;
-  color:#7253ed;
+  color:#var(--color-text-anchor);
+}
+
+.rst-content table.docutils td,
+.rst-content table.docutils th,
+.rst-content table.field-list td,
+.rst-content table.field-list th,
+.wy-table td,
+.wy-table th {
+  padding: 8px 14px;
+}
+
+.dark-mode-toggle {
+  position: absolute;
```

Comment:
if you think this would be useful I would make a PR on this with position "fixed", then it will stay there in the corner even if you scrolled down...